scss/components/_accordion.scss

```diff
@@ -65,24 +67,30 @@ $accordion-content-padding: 1rem !default;
 }
 
 /// Adds styles for the title of an accordion item. Apply this to the link within an accordion item.
-@mixin accordion-title {
+@mixin accordion-title (
+  $padding: $accordion-item-padding,
+  $font-size: $accordion-title-font-size,
+  $color: $accordion-item-color,
+  $border: $accordion-content-border,
+  $background-hover: $accordion-item-background-hover
+) {
   display: block;
-  padding: $accordion-item-padding;
+  padding: $padding;
   line-height: 1;
-  font-size: $accordion-title-font-size;
-  color: $accordion-item-color;
+  font-size: $font-size;
+  color: $color;
```

Comment:
I avoided `-light` and `-dark` because the default button and label do not use the `$pick-best-contrast` function. You might make an app with a dark background and a bright default button with dark text. If necessary, the buttons created from the palette use the alt color, which might be light.